src/components/CommentList/CommentList.scss

```
@@ -1 +1,7 @@
-//
+.CommentList {
+  display: flex;
+  flex-direction: column;
+  gap: 0.7em;
+  background-color: #eee;
```

create variables for colors
```suggestion
  background-color: $c-someColor;
```